File: MusicTime.Web/music-time/src/Models/AlbumDto.ts
class AlbumDto {
  id: number;
  artistName: string;
  title: string;
  genre: string | null;
  description: string | null;
  releaseYear: number | null;
  coverGuid: string | null;

  constructor(
    id: number,
    artistName: string,
    title: string,
    genre: string | null,
    description: string | null,
    releaseYear: number | null,
    coverGuid: string | null
  ) {
    this.id = id;
    this.artistName = artistName;
    this.title = title;
    this.genre = genre;
    this.description = description;
    this.releaseYear = releaseYear;
    this.coverGuid = coverGuid;
  }
}

export default AlbumDto;

File: MusicTime.Bll/Dtos/AlbumDto.cs
using System;

namespace MusicTime.Bll.Dtos
{
    public class AlbumDto
    {
        public int Id { get; set; }
        public string? ArtistName { get; set; }
        public string Title { get; set; }
        public string Genre { get; set; }
        public string Description { get; set; }
        public int? ReleaseYear { get; set; }
        public Guid? CoverGuid { get; set; }
    }
}
